app/root.jsx

```diff
@@ -28,7 +28,21 @@ export default function App() {
       <head>
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <meta name="title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta name="description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="og:type" content="website"/>
+        <meta property="og:url" content="https://getalby.com/"/>
+        <meta property="og:title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta property="og:description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="og:image" content={AlbyHeadIcon}/>
+        <meta property="twitter:card" content="summary_large_image"/>
+        <meta property="twitter:url" content="https://getalby.com/"/>
+        <meta property="twitter:title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta property="twitter:description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
+        <meta property="twitter:image" content={AlbyHeadIcon}/>
```

Comment:
damn, now I was too fast in merging.... those do not create full URLs with host (only `/images/foo.jpg` but we need here `https://getalby.com/images/foo.jpg` otherwise the tools that look at those meta tags can not find the image.

Reply:
Ok I will open a new PR.